library/src/main/java/com/pokegoapi/util/Signature.java

```diff
@@ -38,15 +36,33 @@ public static void setSignature(PokemonGo api, RequestEnvelopeOuterClass.Request
 		SignatureOuterClass.Signature.Builder sigBuilder = SignatureOuterClass.Signature.newBuilder()
 				.setLocationHash1(getLocationHash1(api, authTicketBA, builder))
 				.setLocationHash2(getLocationHash2(api, builder))
-				.setUnk22(ByteString.copyFrom(uk22))
+				.setUnknown22(ByteString.copyFrom(uk22))
 				.setTimestamp(api.currentTimeMillis())
-				.setTimestampSinceStart(curTime - api.startTime);
+				.setTimestampSinceStart(curTime - api.startTime)
+				.setActivityStatus(
```

Comment:
I will remove the activity Status. Since the last update it's not working for me.